### ISSUE TITLE
Compatibility issue with latest pytest versions

### ISSUE THREAD

**Issue by soulrebel** [pytest-dev/pytest-bdd#224]
Since pytest 3.1.0 (currently at 3.2.3) pytest-bdd breaks in [cookiecutter-qa](https://github.com/tierratelematics/cookiecutter-qa/pull/164) with:
```
_pytest.vendored_packages.pluggy.PluginValidationError: unknown hook 'pytest_bdd' in plugin <module 'project_qa.tests.functional.conftest' (<_pytest.assertion.rewrite.AssertionRewritingHook object at 0x7f035b2d4f28>)>
```
Up to version 3.0.7 this does not happen.

**Comment by The-Compiler** [comment]
This doesn't seem to be a pytest-bdd issue - you [import `pytest_bdd` in your `conftest.py`](https://github.com/tierratelematics/cookiecutter-qa/blob/916df0a10f8a934a655b33b8de01d788e851fbb0/%7B%7Bcookiecutter.project_slug%7D%7D/%7B%7Bcookiecutter.project_slug%7D%7D/tests/functional/conftest.py#L4), and apparently `pluggy` (part of pytest for plugin hooks) interprets that as a pytest hook.

As a workaround you can do `import pytest_bdd as bdd`, and I opened an issue here: https://github.com/pytest-dev/pluggy/issues/91

**Comment by RonnyPfannschmidt** [comment]
this will be fixed in the next feature release of pytest (which no longer vendors pluggy so the fix that was implemented for pytest-dev/pluggy#17 can be used)
